src/jolly_brancher/main.py

```diff
@@ -197,15 +197,18 @@ def main(args):
     ticket = args.ticket.upper()
     myissue = jira.issue(ticket)
 
-    summary = myissue.fields.summary
+    summary = myissue.fields.summary.lower()
     summary = summary.replace(" ", "-")
     for bad_char in ["."]:
         summary = summary.replace(bad_char, "")
 
-    issue_type = str(myissue.fields.issuetype)
-    issue_type = issue_type.upper()
+    issue_type = str(myissue.fields.issuetype).lower()  # should issue_type be upper() or lower()?
```

Comment:
I could be persuaded, but I think that I prefer uppercase as it makes a more clear visual delineation.

Reply:
Uppercase works!